waves/timelock/test/timelock.ride-test.js

```diff
@@ -1,260 +1,249 @@
-const { height, waitUntilBlock } = require('./localNode');
+const { height, waitUntilBlock, accountData } = require('./localNode');
+
 const wvs = 10 ** 8;
 
-describe('Timelock test suite', async function() {
-    let lockBlock = 0;
-    const totalChunks = 20;
-    const interval = 2;
-    const initialDelay = 2;
-
-    this.timeout((totalChunks + interval + initialDelay) * 21000);
-
-    before(async () => {
-        await setupAccounts({
-            external: 42 * wvs,
-            script: 42 * wvs,
-            owner: 42 * wvs
-        });
-        
-        const script = compile(file('timelock.ride'));
-        const ssTx = setScript({script}, accounts.script);
-        await broadcast(ssTx);
-        await waitForTx(ssTx.id)
-        console.log('Script has been set')
+describe('Timelock test suite', async function () {
+  let lockBlock = 0;
+  const totalChunks = 4;
+  const interval = 2;
+  const initialDelay = 2;
+
+  this.timeout((totalChunks + interval + initialDelay) * 21000);
+
+  before(async () => {
+    await setupAccounts({
+      external: 42 * wvs,
+      script: 42 * wvs,
+      owner: 42 * wvs,
+    });
+
+    const script = compile(file('timelock.ride'));
+    const ssTx = setScript({ script }, accounts.script);
+    await broadcast(ssTx);
+    await waitForTx(ssTx.id);
+    console.log('Script has been set');
+  });

```

Comment:
This console.log is not really required for unit testing.

Reply:
Since the testing suite is reliant on external services, like a local private node running, and a lot of async functions affected by those external services, it felt important to add that reporting to the test logs so its easy to tell everything is working